### PR TITLE
Feature/test document save

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: tests/qunit/ajax/setup-ajax.js [hunk 1]
```diff
@@ -36,6 +38,7 @@ export const handleSend = ( params ) => {
 				},
 			} );
 		} else {
+			debugger;
 			params.error( `One of the mock actions is missing, please check 'setup-ajax.js'` );
```
Comment (iNewLegend) [inline tests/qunit/ajax/setup-ajax.js]: Remove

File: tests/qunit/ajax/save-builder/mock/autosave.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (iNewLegend) [inline tests/qunit/ajax/save-builder/mock/autosave.json]: try optimize

File: tests/qunit/ajax/save-builder/mock/pending.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (iNewLegend) [inline tests/qunit/ajax/save-builder/mock/pending.json]: try optimize

File: tests/qunit/ajax/save-builder/mock/private.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (iNewLegend) [inline tests/qunit/ajax/save-builder/mock/private.json]: try optimize

File: tests/qunit/ajax/save-builder/mock/publish.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (iNewLegend) [inline tests/qunit/ajax/save-builder/mock/publish.json]: try optimize.

File: tests/qunit/ajax/setup-ajax.js [hunk 6]
```diff
@@ -27,12 +30,17 @@ export const handleSend = ( params ) => {
...
-		} );
+		if ( Object.entries( responses ).length === Object.entries( params.data.actions ).length ) {
+			params.success( {
+				success: true,
+				data: {
+					responses,
+				},
+			} );
+		} else {
+			debugger;
```
Comment (iNewLegend) [inline tests/qunit/ajax/setup-ajax.js]: remove

File: tests/qunit/ajax/save-builder/index.js [hunk 7]
```diff
@@ -5,6 +5,18 @@ export const saveBuilder = ( action, fullParams ) => {
 		case 'draft':
 			return mock.draft;
 
+		case 'autosave':
```
Comment (matipojo) [inline tests/qunit/ajax/save-builder/index.js]: `return mock[ action.data.status ] || { success: false, code: 500 };`?

**Review by matipojo — APPROVED** [review]

**Pull request merged**